package-lock.json

```diff
@@ -13903,28 +13903,28 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": false,
+          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
Revert this file?

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/1468/commits/e6b1a00aabf452ee071c0c1b63dabcc847134cee